src/auth/authClient.tsx

```diff
@@ -41,6 +47,17 @@ const signup: (user: SignupRequest) => Promise<TokenPayload> = (
     (response) => response.data,
   );
 
+const setContacts: (
+  contactInfo: SetContactsRequest,
+  accessToken: string,
```

Comment:
Now that we're moving this to its own page and submission, you should not have to include the accessToken